### DISCUSSION TITLE
Option to show a red border around NFSW images and content

### DISCUSSION THREAD

**Discussion by no-connections** [civitai/civitai#237] [Ideas]
Right now there is no way to know if an image or model is tagged NSFW without opting to hide NSFW content and see if it got blurred.
I don't want to hide NSFW, but I want to know if it is NSFW. Especially if I want to link a model to someone else. This will also help with spotting images not tagged properly, since now only those that hide NSFW will know.

**Comment by edgartaor** [comment]
>I don't want to hide NSFW, but I want to know if it is NSFW.

Oh I just submit a similar one
https://github.com/civitai/civitai/discussions/244

**Comment by manuelurenah** [comment]
Hey @no-connections! This has already been implemented, you should be able to see which models are labeled as NSFW (18+) even if you have opted-out from blurring this kind of content from your account settings. Thanks again for bringing this up 🙌
